.github/workflows/pull_request_release.yml

```diff
@@ -24,7 +25,10 @@ jobs:
           NODE_ENV=development npm ci
           git config --local user.email "one.amex@aexp.com"
           git config --local user.name "OneAmexBot"
-          npm run release
+          npm run release -- --no-verify
```

Comment:
what does the `--no-verify` flag do?

Reply:
This would skip the hooks as we don't need this in this case

Reply:
see my below comment in package.json